p2p/p2p_test.go

```diff
@@ -17,25 +17,18 @@
 package p2p
 
 import (
-	"fmt"
-	"io/ioutil"
 	"math/big"
-	"os"
-	"path"
 	"reflect"
 	"testing"
 	"time"
 
 	"github.com/ChainSafe/gossamer/common"
 	"github.com/ChainSafe/gossamer/common/optional"
-	net "github.com/libp2p/go-libp2p-core/network"
 	peer "github.com/libp2p/go-libp2p-core/peer"
-	ps "github.com/libp2p/go-libp2p-core/peerstore"
-	ma "github.com/multiformats/go-multiaddr"
 )
 
-func startNewService(t *testing.T, cfg *Config, msgSendChan chan []byte, msgRecChan chan BlockAnnounceMessage) *Service {
-	node, err := NewService(cfg, msgSendChan, msgRecChan)
+func startNewService(t *testing.T, cfg *Config, sendChan chan []byte, recChan chan BlockAnnounceMessage) *Service {
```

Comment:
I think you removed the wrong part, having `chan` in the name is a cardinal sin.

Reply:
Ha! Noted and updated to `msgSend` and `msgRec`.